services/ingress-nginx/values-base.yaml

```diff
@@ -0,0 +1,21 @@
+ingress-nginx:
+  controller:
+    config:
+      compute-full-forwarded-for: "true"
+      large-client-header-buffers: "4 64k"
+      proxy-body-size: "100m"
+      proxy-buffer-size: "64k"
+      ssl-redirect: "true"
+      use-forwarded-headers: "true"
+    service:
+      externalTrafficPolicy: Local
+      loadBalancerIP: "139.229.146.150"
+    podLabels:
+      hub.jupyter.org/network-access-proxy-http: "true"
+
+vault_certificate:
+  enabled: false
+
+pull-secret:
+  enabled: true
+  path: secret/k8s_operator/bleed.lsst.codes/pull-secret
```

Comment:
bleed / base?

Reply:
Oh, good, catch, thank you! Fixed both of those.